### PR TITLE
test(x/ecocredit): marketplace update acceptance tests

### PR DESCRIPTION
**Pull request by ryanchristo**
## Description

Closes: #894

This pull request adds marketplace acceptance tests for updating a sell order.

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] followed the guidelines for [building modules](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules)
- [ ] included the necessary unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed 
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: x/ecocredit/marketplace/msg_update_sell_orders.go [hunk 1]
```diff
@@ -24,26 +26,55 @@ func (m MsgUpdateSellOrders) GetSignBytes() []byte {
 
 // ValidateBasic does a sanity check on the provided data.
 func (m *MsgUpdateSellOrders) ValidateBasic() error {
+	if len(m.Owner) == 0 {
+		return sdkerrors.ErrInvalidRequest.Wrap("owner cannot be empty")
+	}
+
 	if _, err := sdk.AccAddressFromBech32(m.Owner); err != nil {
-		return sdkerrors.ErrInvalidAddress
+		return sdkerrors.ErrInvalidRequest.Wrapf("owner is not a valid address: %s", err)
```
Comment (technicallyty) [inline x/ecocredit/marketplace/msg_update_sell_orders.go:34]: i think we opted to keep ErrInvalidAddress as we did in #1140

**Review by technicallyty — APPROVED** [review]
lgtm!

File: x/ecocredit/marketplace/msg_update_sell_orders.go (hunk 1 above)
Comment (ryanchristo) [reply x/ecocredit/marketplace/msg_update_sell_orders.go]: https://github.com/regen-network/regen-ledger/pull/1134/commits/6b5d995e5a77025f4155f0e081056b0152cff09f

**Review by aleem1314 — APPROVED** [review]
lgtm

**Pull request merged**